manifests/image-references

```diff
@@ -66,3 +66,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-thanos:latest
+  - name: console-plugin
+    from:
+      kind: DockerImage
+      name: quay.io/anpicker/monitoring-plugin:4-13-test
```

Comment:
@simonpasquier do you know if these must always refer to quay.io/openshift? Since we are missing this image, I wonder if this would work to `launch <pr>` ?

Reply:
Resolving this. This can't be done until the console-plugin image is published to `quay.io/openshift` repo. `ci/images` also fails for the same reason. OpenShift CI  only loads know images.

cc: @kyoto